dts/client.py

```diff
@@ -22,8 +21,9 @@ def __init__(self, api_key):
         self.api_key = api_key
 
     def __call__(self, r):
```

Comment:
I'd err on the side of clarity and call the variable `request` (assuming that's what it is)